models/reviews.models.js

```diff
@@ -34,6 +34,23 @@ exports.updateReviewById = (review_id, inc_votes = "undefined") => {
     });
 };
 
+exports.fetchCommentsByReviewId = (review_id) => {
+  return db
+    .query(
+      `SELECT * FROM comments WHERE review_id=$1 ORDER BY created_at DESC`,
+      [review_id]
+    )
+    .then(({ rows: comments }) => {
+      if (!comments) {
```

Comment:
rows is always truthy so this code will never run - tests will still be passing though as you are handling this in the controller